src/main/java/lotto/domain/LottoGenerator.java

```diff
@@ -4,24 +4,28 @@
 import lotto.domain.game.LottoNumberSet;
 import lotto.domain.game.LottoNumber;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 import java.util.stream.Stream;
 
+import static java.util.stream.Collectors.toList;
+
 /**
  * Created By mand2 on 2020-11-19.
  * 로또 번호 6개 자동 생성하는 객체
  */
 public class LottoGenerator {
 
     private static List<Integer> lottoNumbers;
```

Comment:
```suggestion
    private static List<LottoNumber> lottoNumbers = IntStream.rangeClosed(LottoNumber.VALID_MIN_NUMBER, LottoNumber.VALID_MAX_NUMBER)
                .mapToObj(LottoNumber::new)
                .collect(toList());
```

LottoNumber 클래스가 생겼으니 상수풀을 LottoNumber 객체로 가질 수 있게되었네요 :)